.travis.yml

```diff
@@ -8,51 +8,104 @@ addons:
+    - TEST_SUITE=functional MFTF_SUITE=adobe_stock_integration_suite_configuration
+    - TEST_SUITE=functional MFTF_SUITE=adobe_stock_integration_suite_keywords
+    - TEST_SUITE=functional MFTF_SUITE=adobe_stock_integration_suite_ims
+    - TEST_SUITE=functional MFTF_SUITE=adobe_stock_integration_suite_see_more
 matrix:
   fast_finish: true
+jobs:
+  # only run MFTF tests on PHP 7.3
+  exclude:
+    - php: '7.1'
+      env: TEST_SUITE=functional MFTF_SUITE=adobe_stock_integration_suite_filters_part_one
+    - php: '7.1'
+      env: TEST_SUITE=functional MFTF_SUITE=adobe_stock_integration_suite_filters_part_two
+    - php: '7.1'
+      env: TEST_SUITE=functional MFTF_SUITE=adobe_stock_integration_suite_grid
+    - php: '7.1'
+      env: TEST_SUITE=functional MFTF_SUITE=adobe_stock_integration_suite_preview
+    - php: '7.1'
+      env: TEST_SUITE=functional MFTF_SUITE=adobe_stock_integration_suite_configuration
+    - php: '7.1'
+      env: TEST_SUITE=functional MFTF_SUITE=adobe_stock_integration_suite_keywords
+    - php: '7.1'
+      env: TEST_SUITE=functional MFTF_SUITE=adobe_stock_integration_suite_ims
+    - php: '7.1'
+      env: TEST_SUITE=functional MFTF_SUITE=adobe_stock_integration_suite_see_more
+    - php: '7.2'
+      env: TEST_SUITE=functional MFTF_SUITE=adobe_stock_integration_suite_filters_part_one
+    - php: '7.2'
+      env: TEST_SUITE=functional MFTF_SUITE=adobe_stock_integration_suite_filters_part_two
+    - php: '7.2'
+      env: TEST_SUITE=functional MFTF_SUITE=adobe_stock_integration_suite_grid
+    - php: '7.2'
+      env: TEST_SUITE=functional MFTF_SUITE=adobe_stock_integration_suite_preview
+    - php: '7.2'
+      env: TEST_SUITE=functional MFTF_SUITE=adobe_stock_integration_suite_configuration
+    - php: '7.2'
+      env: TEST_SUITE=functional MFTF_SUITE=adobe_stock_integration_suite_keywords
+    - php: '7.2'
+      env: TEST_SUITE=functional MFTF_SUITE=adobe_stock_integration_suite_ims
+    - php: '7.2'
+      env: TEST_SUITE=functional MFTF_SUITE=adobe_stock_integration_suite_see_more
 cache:
   directories:
     - $HOME/.composer/cache
     - $HOME/.nvm
     - $HOME/node_modules
     - $HOME/yarn.lock
+    # cache location of webdrivers for MFTF (chromedriver, geckdriver, etc)
+    - $HOME/drivers
 before_install:
-  - mkdir -p ext/magento/adobe-stock-integration
-  - mv !(ext) ext/magento/adobe-stock-integration/
-  - ./ext/magento/adobe-stock-integration/dev/travis/before_install.sh
+  - bash -x dev/travis/before_install.sh
 install:
-  - cd magento2
+  - mkdir -p magento2/app/code/Magento
+  - mv Adobe* magento2/app/code/Magento/.
+  - pushd magento2
   - composer install
-  - composer config minimum-stability dev
-  - composer config repositories.ext path "../ext/*/*/*"
-  - composer require magento/adobe-stock-integration
+  - composer require astock/stock-api-libphp
+  - popd
 before_script:
-  - cp ../ext/magento/adobe-stock-integration/dev/tests/unit/phpunit.xml dev/tests/unit/phpunit.xml
```

Comment:
It would be better to copy our custom `phpunit.xml` to the default location for easier unit tests launch